applications/simple_radar_pipeline/python/simple_radar_pipeline.py

```diff
@@ -237,4 +237,4 @@ def compose(self):
 
     duration = (iterations * num_pulses * num_channels) / (tstop - tstart)
 
-    print(f"{duration:0.3f} pulses/channel/sec")
+    print(f"{duration:0.3f} pulses/sec")
```

Comment:
The readme says "The output of this demonstration is a measure of the number of pulses per channel per second processed on GPU"

Why is the per channel removed?

Reply:
Updated the readme. I wanted to show aggregate since the throughput is much higher. The previous printout was a bug since it was not per channel.

Reply:
Is this only done for python? I still see the ` 35648 pulses/channel/sec` in the cpp version.

Reply:
This should be fixed now. Thanks for catching that!